server/routes/organization.js

```diff
@@ -37,14 +37,15 @@ router.route('/add').post((req, res) => {
     const location = req.body.location;
     const orgPhone = req.body.orgPhone;
     const orgEmail = req.body.orgEmail;
+    const orgType = req.body.orgType;
     //The following three variables represent the tables for shifts, admins and volunteers of an organization.
     const admins = req.body.admins;
     const shifts = req.body.shifts;
     const volunteers = req.body.volunteers;
 
     console.log(req.body);
 
-    const newOrg = new Organization({orgName, adminUsername, location, orgPhone, orgEmail, shifts, volunteers, admins});
+    const newOrg = new Organization({orgName, adminUsername, orgType, location, orgPhone, orgEmail, shifts, volunteers, admins});
```

Comment:
`shifts and volunteers`  is probably not necessary when creating an organization. I don't think it will affect it since it will just be null/undefined though.

Reply:
I put those tables in here to ensure those fields will be in the database (even though yes they are empty upon creation), but now that I think about it they probably are added regardless since it creates the object based on the schema. I'll test that and update you